src/ble.md

```diff
@@ -214,7 +215,38 @@ Set Hostname:
 |      | bytes of hostname  |
 | CS   | checksum           |
 
-This command will trigger one RPC Response which will contain the hostname of the device.
+This command will trigger one RPC Response which will contain the hostname of the device. Getting or setting this
+property should reset the authorization timeout.
+
+### RPC Command: Get/Set Device Name
+
+Sends a request for the device to either get or set its name. This could mean different things depending on the device
+manufacturer.  It may alter the default "hostname" or not. If setting both this property and hostname, it is recommended
+to set hostname first then device name. It should generally accept any UTF-8 encoded string however some device 
+manufacturers may choose not to support characters outside the ASCII range.  Getting this property should  
+(unless it contains non-ASCII characters) return the same value as the Device Info's "Device Name" (4th) property.
```

Comment:
> Getting this property should (unless it contains non-ASCII characters) return the same value as the Device Info's "Device Name" (4th) property.

Why does the character set matter to the response of this command vs that of device info? The character encoding of the device info is not specified, should we do that?
Maybe specify strings sent from the device are either 7-bit ascii or UTF-8 (UTF-8 is backwards compatible with 7-bit ascii).
If the device does not accept UTF-8 in strings sent to the device, I think we need a flag for that.

My preference would be to simply specify all strings are UTF-8 (with restrictions for codes where applicable, such as for the hostname).

Reply:
Should we say manufacturers must support 7-bit ASCII and may choose to support the full UTF-8 character set? Or just say all strings are UTF-8 encoded and the full UTF character set must be supported for device name?